pkg/util/exec/exec.go

```diff
@@ -198,8 +204,17 @@ func logOutput(name string, stdout, stderr io.ReadCloser) {
 		}
 	}
 
+	go func() {
+		in := bufio.NewScanner(stderr)
+		lastLine := ""
```

Comment:
you might consider a helper that reads from a param of type reader. Then you could just call 
```
go logFromReader(stderr)
logFromReader(stdout)
```